Upgrade tutorial for 12.1.0.Final (#5)
infinispan/infinispan-server-tutorial: README.adoc

* Pull the container image and run with https://www.docker.com/[Docker] or https://podman.io/[Podman].
* Download the server distribution and extract it to your filesystem.

[IMPORTANT]

> .Credentials

Reviewer: ```suggestion
```

> By default, Infinispan Server *requires user authentication* and *authorization* is implicit since

Reviewer: ```suggestion
By default, Infinispan Server requires user *authentication* and *authorization*.
```

> 12.1. *Creating a user named `admin` gives you administrative access to Infinispan Server.*

Reviewer: ```suggestion
For the purposes of this tutorial, create a user named `admin` so you get administrative access to Infinispan Server.
```
